### PR TITLE
Fix a couple small errors

### PR DESCRIPTION
**Pull request by elcorbs**
- Fixed a type in convertors.
- Removed unnecessary dependencies causing a validation error.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ services:
       S151_TAB: 'yes'
       RM_MONTHLY_CATCHUP_TAB: 'yes'
       OUTPUTS_ACTUALS_TAB: 'yes'
+      NO_LOGS: 'true'
```
Comment (ccp92) [inline docker-compose.yml]: Should probably take this back out again before merging into master

Comment (elcorbs) [inline docker-compose.yml]:
```suggestion
```

File: lib/common/domain/template.rb [hunk 2]
```diff
@@ -28,6 +28,7 @@ def get_invalid_paths(schema, project_data)
     nil
   rescue JsonSchema::AggregateError => aggregated_errors
     get_supported_errors(aggregated_errors).map do |error|
+      pp error
```
Comment (ccp92) [inline lib/common/domain/template.rb]: Should take out the pp

Comment (elcorbs) [inline lib/common/domain/template.rb]:
```suggestion
```

**Pull request merged**